MeetupPlatformApi/Features/Meetups/CancleSignUpToMeetup/CancelSignUpToMeetupFeature.cs

```diff
@@ -0,0 +1,62 @@
+﻿namespace MeetupPlatformApi.Features.Meetups.CancleSignUpToMeetup;
+
+using MeetupPlatformApi.Authentication.Helpers;
+using MeetupPlatformApi.Persistence.Context;
+using MeetupPlatformApi.Seedwork.WebApi;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.EntityFrameworkCore;
+
+[ApiSection(ApiSections.Meetups)]
+public class CancelSignUpToMeetupFeature : FeatureBase
+{
+    private readonly ApplicationContext context;
+
+    public CancelSignUpToMeetupFeature(ApplicationContext context)
+    {
+        this.context = context;
+    }
+
+    /// <summary>
+    /// Cancel sign up to meetup by its id.
+    /// </summary>
+    /// <response code="404">If needed meetup is null.</response>
+    /// <response code="400">If user doesn't exist or the user is trying to sign up for meetup twice.</response>
+    /// <response code="204">If there's not errors after executing method.</response>
+    [HttpDelete("/api/meetups/cancel-sign-up/{id:guid}")]
+    [Authorize(Roles = Roles.PlainUser)]
+    [ProducesResponseType(StatusCodes.Status404NotFound)]
+    [ProducesResponseType(StatusCodes.Status400BadRequest)]
+    [ProducesResponseType(StatusCodes.Status204NoContent)]
+    public async Task<IActionResult> CancelSignUpToMeetup([FromRoute] Guid id)
```

Comment:
In this method You have used an `.Include` for both `meetup` and `user`. Please, remove one of these includes and use the other one